lib/std/os/linux.zig

```diff
@@ -2054,10 +2054,10 @@ pub fn eventfd(count: u32, flags: u32) usize {
     return syscall2(.eventfd2, count, flags);
 }
 
-pub fn timerfd_create(clockid: clockid_t, flags: TFD) usize {
```

Comment:
I haven't looked at the PR in a few days so I might be missing some context here - why are we changing this to an integer?

Reply:
I based my work on the kernel sources, there it is an integer.

Reply:
I didn't mean it *quite* that literally when I said the kernel code should be the basis for our decision here. 🙂 We do use enums in syscall wrappers when the set of acceptable values is known ahead of time - which I think is the case here?

Reply:
Was unsure but can change that.